lighthouse-core/gather/gatherers/trace.js

```diff
@@ -51,9 +51,12 @@ class Trace extends FRGatherer {
       // This doesn't add its own events, but adds a `stackTrace` property to devtools.timeline events
       'disabled-by-default-devtools.timeline.stack',
 
-      // CPU sampling profiler data only enabled for debugging purposes
-      // 'disabled-by-default-v8.cpu_profiler',
-      // 'disabled-by-default-v8.cpu_profiler.hires',
+      // Additional categories used by devtools. Not used by Lighthouse, but included to facilitate
+      // loading traces from Lighthouse into the Performance panel.
+      'disabled-by-default-devtools.timeline.frame',
+      'disabled-by-default-v8.cpu_profiler',
+      'disabled-by-default-v8.cpu_profiler.hires',
+      'latencyInfo',
```

Comment:
how much larger do these typically make traces? Do we need to update things like [`minify-trace`](https://github.com/GoogleChrome/lighthouse/blob/0de46e7c9843cb17c905be2431aec7dd4ffb7e73/lighthouse-core/lib/minify-trace.js) to strip new things (or include new things)?

cc @patrickhulce

Reply:
I would expect `cpu_profiler.hires` to be the largest of these by far on busy pages (which we were planning on enabling anyhow #8526). I think `latencyInfo` can be fairly chatty too if it's what I think it is.

Updating minify-trace to keep what we need from here, would be great! Could use that process to get some rough data on trace sizes for our common culprits (cnn.com, sfgate.com, theverge.com)